drivers/CoreVideo/include/CoreDSI.hpp

```diff
@@ -4,15 +4,15 @@
 
 #pragma once
 
-#include "CoreSTM32HalBase.h"
 #include "interface/DSI.hpp"
+#include "interface/drivers/STM32Hal.h"
 
 namespace leka {
 
 class CoreDSI : public interface::DSIBase
```

Comment:
Pour les mêmes raisons que `CoreSTM32HalBase` devient `interface::STM32Hal`, on peut avoir `interface::DSIBase` qui devient juste `interface::DSI`

```suggestion
class CoreDSI : public interface::DSI
```